SingularityService/src/test/java/com/hubspot/singularity/SingularityAuthorizationHelperTest.java

```diff
@@ -2,10 +2,10 @@
 
 import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertTrue;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.*;
```

Comment:
Last thing on this then we can merge @darcatron . We try to avoid any import * when possible. Can you split this out?